backend/src/gpml/constants.clj

```diff
@@ -1,10 +1,12 @@
 (ns gpml.constants)
 
-(def resource-types ["financing_resource" "technical_resource" "action_plan"])
+(def resource-types #{"financing_resource" "technical_resource" "action_plan"})
```

Comment:
just a doble check here... are we returning this data directly to the FE? in that case do we need order there? in that case 😄 perhaps we need 2 defs, `(def resource-types ["financing_resource" "technical_resource" "action_plan"])` and `(def resource-types-set (set resource-types))`

Reply:
I don't think we are returning it to the FE directly. I did consider adding a new constant, but it seems like the main use is to check containership, and one use where we do the same thing in a DB query.